### ISSUE TITLE
Screen sharing

### ISSUE THREAD

**Issue by gymnae** [robertoandrade/docker-jitsi-meet#5]
Hey there,
great container so far, I'm even getting it to work with [let'sencrypt ssl proxy companion](https://github.com/JrCs/docker-letsencrypt-nginx-proxy-companion) for a [nginx proxy](https://github.com/jwilder/nginx-proxy) - Working as in people can join a channel, but voice and audio are not working.

I'm running the image on a CoreOS bare-metal host. To make it work, I adjust the sites-enabled config to not load in additional SSL certs and not listen on 443, as the ssl proxy takes care of that. I tried to pinpoint is this is the source of the problem by forwarding 443 to the IP of the server and connect to that, but it's the same issue

The desired feature doesn't work, which works with the jitsi-meet test on their official server:
The screen sharing plugin won't install or work.
Any idea why that may be?

In short: The container starts, letsencrypt certs are generated and the connection is SSL, people can join and create channels, but there is no audio/video, etherpad or screen sharing

Cheers

**Comment by erickeller** [comment]
same issue here. I just run it locally and use iptable to forward the hits to the container:

```
#!/bin/sh
TARGET_IP=172.17.0.2
INTERFACE=wlo1
test -z $1 || TARGET_IP=$1
test -z $2 || INTERFACE=$2
iptables -t nat -A PREROUTING -i ${INTERFACE} -p tcp --dport 5347 -j DNAT --to ${TARGET_IP}:5347
iptables -t nat -A PREROUTING -i ${INTERFACE} -p tcp --dport 80 -j DNAT --to ${TARGET_IP}:80
iptables -t nat -A PREROUTING -i ${INTERFACE} -p tcp --dport 443 -j DNAT --to ${TARGET_IP}:443
iptables -t nat -A PREROUTING -i ${INTERFACE} -p udp --dport 10000:10010 -j DNAT --to ${TARGET_IP}:10000-10010
```

I can join the room but no audio, no video...

**Comment by lokeshh** [comment]
Same here... no audio/video

**Comment by camilotorres97** [comment]
Same problem here, someone fixed it?
